src/app/contextBroker/CMakeLists.txt

```diff
@@ -88,6 +88,9 @@ ELSEIF(${DISTRO} STREQUAL "openSUSE_13.1")
 ELSEIF(${DISTRO} MATCHES "CentOS_6.*")
   MESSAGE("contextBroker: CentOS DISTRO: '${DISTRO}'")
   TARGET_LINK_LIBRARIES(contextBroker ${STATIC_LIBS} -lmicrohttpd -lmongoclient -lboost_thread-mt -lboost_filesystem-mt -lboost_system-mt -lboost_regex-mt)
+ELSEIF(${DISTRO} MATCHES "CentOS_release")
```

Comment:
Yes, me too :)

But with "CentOS_7.*" did not work and the console displays "CentOS_release" as version.
-> cat /etc/redhat-release
```
CentOS Linux release 7.2.1511 (Core) 
```
-> scripts/build/osDistro.sh
```
CentOS_release
``` 

With this failover code it's solved:
```
centos_distro=$(cat /etc/redhat-release 2> /dev/null | awk '{print $3}') 
if [ $centos_distro == "release" ]; then
        centos_distro=$(cat /etc/redhat-release 2> /dev/null | awk '{print $4}')
fi
```